nixpkgs_review/nix.py

```diff
@@ -172,9 +269,79 @@ def nix_build(
     ] + shlex.split(args)
 
     try:
-        sh(command)
-    except subprocess.CalledProcessError:
-        pass
+        proc = sh(command, stderr=subprocess.PIPE)
+        stderr = proc.stderr
+    except subprocess.CalledProcessError as e:
+        stderr = e.stderr
+
+    # Remove boring 'copying path' lines from stderr
+    stderr = "\n".join(
+        line
+        for line in stderr.splitlines()
+        if not (
+            line.startswith("copying path '")
+            and line.endswith("from 'https://cache.nixos.org'...")
```

Comment:
This does not work for cachix or nix-serve which are used by Mic92's build farm.

Reply:
I can remove all "copying path '" from the logs. It's just for convenience though.